core/client/build.gradle

```diff
@@ -19,6 +19,7 @@ dependencies {
     annotationProcessor "org.projectlombok:lombok:1.18.30"
     testAnnotationProcessor "org.projectlombok:lombok:1.18.30"
 
+    testImplementation project(":administrative")
```

Comment:
reminder to remove

Reply:
This is being used on the unit tests, do you see any problem on that ?

Reply:
conceptually we shouldn't have this dependency (even for tests) would prefer to copy the test models over.

Reply:
I get your point, what about this?
https://github.com/thousandeyes/thousandeyes-sdk-java/pull/89/commits/75b9f7fd65d7eba263ac6eb2fdb6f2236344a108

Reply:
even better 😃 👍